.ci/scripts/distribution/ensure-naming-for-process.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -eu
 
-FIND_OUTPUT=$(find . -iname '*workflow*' -not -path '.*/node_modules/*' -not -path '.*/target/*' -not -path '.*/.git/*'  -not -path '.*/.github/*' -not -path '.*/vendor/*')
+FIND_OUTPUT=$(find . -iname '*workflow*' -not -path '.*/node_modules/*' -not -path '.*/target/*' -not -path '.*/.git/*'  -not -path '.*/.github/*' -not -path '.*/vendor/*' -not -path '.*/docker/*')
 
 # if grep doesn't find a match it exits with 1, which we ignore as we check the output later
-GREP_OUTPUT=$(grep --exclude-dir={.git,node_modules,target,.ci,.github,vendor} -i -P 'workflow(?![-\s]+(?i)engine)' -r . || true)
+GREP_OUTPUT=$(grep --exclude-dir={.git,node_modules,target,.ci,.github,vendor,docker} --exclude-from={Dockerfile} -i -P 'workflow(?![-\s]+(?i)engine)' -r . || true)
```

Comment:
:x: I don't think the `--exlude-from` is correct, it expects a file with a list of exclude globs.

Reply:
It works though :smile: Anyway, you're right, `exclude` is probably more appropriate :+1:

Reply:
do we actually need to exclude the Dockerfile? "workflow" doesn't appear in that file anyway.

Reply:
It does in the tags, line 63 and 64

Reply:
:facepalm: You're right of course.